[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NoClassDefFoundError: java/sql/SQLException
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.internal.util.config.ConfigurationException: Could not locate cfg.xml resource [hibernate.cfg.xml]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.TaskRepository  - Get Task Error: org.hibernate.hql.internal.ast.QuerySyntaxException: TASK is not mapped [SELECT t FROM TASK t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.application.HelloApplication  - Sorry, the main fxml could not be loaded
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.boot.registry.classloading.spi.ClassLoadingException: Unable to load class [bg.tu_varna.sit.library.data.entities.Task]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.UserInfo.User_idUser
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: org.hibernate.hql.internal.ast.QuerySyntaxException: USER is not mapped [SELECT t FROM USER t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.FORM references an unknown entity: bg.tu_varna.sit.library.data.entities.FORM
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.MappingException: Unable to find column with logical name: foreignidForm in org.hibernate.mapping.Table(FORM) and its related supertables and secondary tables
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.USER.UserInfo, referenced property unknown: bg.tu_varna.sit.library.data.entities.UserInfo.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.USER.UserInfo, referenced property unknown: bg.tu_varna.sit.library.data.entities.UserInfo.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Unknown mappedBy in: bg.tu_varna.sit.library.data.entities.UserType.byUSER, referenced property unknown: bg.tu_varna.sit.library.data.entities.USER.idUserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Illegal attempt to map a non collection as a @OneToMany, @ManyToMany or @CollectionOfElements: bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.idUserType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.userType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.status in bg.tu_varna.sit.library.data.entities.Status.byStatus
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Referenced property not a (One|Many)ToOne: bg.tu_varna.sit.library.data.entities.FORM.idForm in mappedBy of bg.tu_varna.sit.library.data.entities.FORM.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: No row with the given identifier exists: [bg.tu_varna.sit.library.data.entities.UserType#0]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType in bg.tu_varna.sit.library.data.entities.UserType.byUSER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.UserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @OneToOne or @ManyToOne on bg.tu_varna.sit.library.data.entities.USER.USERTYPE_idUserType references an unknown entity: bg.tu_varna.sit.library.data.entities.UserType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: mappedBy reference an unknown target entity property: bg.tu_varna.sit.library.data.entities.USER.USERTYEPE_idUserType in bg.tu_varna.sit.library.data.entities.UserType.byType
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - Get user error: org.hibernate.InstantiationException: No default constructor for entity:  : bg.tu_varna.sit.library.data.entities.USER
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.service.spi.ServiceException: Unable to create requested service [org.hibernate.engine.jdbc.env.spi.JdbcEnvironment]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Get user error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.Books] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.USER]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idBook' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Get user error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.Books] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.USER]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idBook' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idBook' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idBook' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.StateOfBooks] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.Books]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.StateOfBooks] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.Books]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Duplicate entry '1' for key 'books.PRIMARY'
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.StateOfBooks] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.Books]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: Type specified for TypedQuery [bg.tu_varna.sit.library.data.entities.StateOfBooks] is incompatible with query return type [class bg.tu_varna.sit.library.data.entities.Books]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Duplicate entry '2' for key 'books.PRIMARY'
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: org.hibernate.hql.internal.ast.QuerySyntaxException: StateOfBooks is not mapped [SELECT t FROM StateOfBooks t]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Duplicate entry '3' for key 'books.PRIMARY'
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.BooksRepository  - Book save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Books entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: org.hibernate.hql.internal.ast.QuerySyntaxException: STATEOFBOOKS is not mapped [SELECT t FROM STATEOFBOOKS t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.StateOfBooksRepository  - Get book state error: org.hibernate.hql.internal.ast.QuerySyntaxException: STATEOFBOOKS is not mapped [SELECT t FROM STATEOFBOOKS t]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.MappingException: Unable to find column with logical name: books_idBook in org.hibernate.mapping.Table(books) and its related supertables and secondary tables
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.Eksemplqri.idBook
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'isbnUnikalenNomer' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errorThe database returned no natively generated identity value
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: @Column(s) not allowed on a @OneToOne property: bg.tu_varna.sit.library.data.entities.Status.status
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User save errornot-null property references a null or transient value : bg.tu_varna.sit.library.data.entities.USER.dateOfUserApproval
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.USER entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idUser' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.USER entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.MappingException: Composite-id class must implement Serializable: bg.tu_varna.sit.library.data.entities.UserInfo
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =: un]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =: un]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =: un]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =u5]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =: un]
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.UserRepository  - User delete errororg.hibernate.QueryException: No data type for node: org.hibernate.hql.internal.ast.tree.IdentNode 
 \-[IDENT] IdentNode: 'username' {originalText=username}
 [select username FROM bg.tu_varna.sit.library.data.entities.USER WHERE username =: un]
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`user`, CONSTRAINT `fk_user_status1` FOREIGN KEY (`status_idStatus`) REFERENCES `status` (`idStatus`))
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedorg.hibernate.AnnotationException: Referenced property not a (One|Many)ToOne: bg.tu_varna.sit.library.data.entities.Status.idStatus in mappedBy of bg.tu_varna.sit.library.data.entities.USER.STATUS_idStatus
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`user`, CONSTRAINT `fk_user_status1` FOREIGN KEY (`status_idStatus`) REFERENCES `status` (`idStatus`))
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`user`, CONSTRAINT `fk_user_status1` FOREIGN KEY (`status_idStatus`) REFERENCES `status` (`idStatus`))
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`userinfo`, CONSTRAINT `fk_userinfo_user1` FOREIGN KEY (`user_idUser`) REFERENCES `user` (`idUser`))
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`userinfo`, CONSTRAINT `fk_userinfo_user1` FOREIGN KEY (`user_idUser`) REFERENCES `user` (`idUser`))
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`userinfo`, CONSTRAINT `fk_userinfo_user1` FOREIGN KEY (`user_idUser`) REFERENCES `user` (`idUser`))
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.access.Connection  - Initial SessionFactory created failedjava.lang.NullPointerException: Cannot invoke "org.hibernate.mapping.PersistentClass.getTable()" because "classMapping" is null
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Field 'idState' doesn't have a default value
[JavaFX Application Thread] ERROR bg.tu_varna.sit.library.data.repositories.EksemplqriRepository  - Eksemplqri save errorcould not execute statement
[JavaFX Application Thread] ERROR org.hibernate.AssertionFailure  - HHH000099: an assertion failure occurred (this may indicate a bug in Hibernate, but is more likely due to unsafe use of the session): org.hibernate.AssertionFailure: null id in bg.tu_varna.sit.library.data.entities.Eksemplqri entry (don't flush the Session after an exception occurs)
[JavaFX Application Thread] ERROR org.hibernate.engine.jdbc.spi.SqlExceptionHelper  - Cannot delete or update a parent row: a foreign key constraint fails (`bookstore`.`eksemplqri`, CONSTRAINT `fk_EKSEMPLQRI_STATEOFBOOKS1` FOREIGN KEY (`STATEOFBOOKS_idState`) REFERENCES `stateofbooks` (`idState`))
